note-app/src/components/DisplayedNote/DisplayedNote.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { styles } from './styles';
+
+const DisplayedNote = ({ activeNote }) => (
+  <div style={styles.wrapper}>
+    {!activeNote ? (
+      <p style={styles.noNoteMessage}>Select note to display</p>
+    ) : (
+      <div>
+        <p style={styles.noteTitle}>{activeNote.title}</p>
+        <p style={styles.noteDescription}>{activeNote.description}</p>
+        <p style={styles.noteCreation}>{activeNote.creation}</p>
+      </div>
+    )}
```

Comment:
Try to avoid using an unnecessary negation, it's can be hard to read the code. No need to make any changes, just note for the future:
```suggestion
    {activeNote ? (
      <div>
        <p style={styles.noteTitle}>{activeNote.title}</p>
        <p style={styles.noteDescription}>{activeNote.description}</p>
        <p style={styles.noteCreation}>{activeNote.creation}</p>
      </div>
    ) : (
      <p style={styles.noNoteMessage}>Select note to display</p>
    )}
```